2200056/10505 소득 불균형.py

```diff
@@ -0,0 +1,11 @@
+T = int(input())
+for t in range(1, T+1):
+    N = int(input())
+    num_list = list(map(int, input().split())) # 인풋값 리스트에 저장
+    num_list_avg = sum(num_list) / len(num_list) # 리스트 평균값 구하기
```

Comment:
깔끔한 풀이군요 👍
`N` 값을 입력을 받았으니, `len`함수를 사용하지 않고 평균을 구하는 방법도 있습니다!
```suggestion
    N = int(input())
    num_list = list(map(int, input().split())) # 인풋값 리스트에 저장
    num_list_avg = sum(num_list) / N # 리스트 평균값 구하기
```